vignettes/manual/SimpleNBox.Rmd

```diff
@@ -0,0 +1,177 @@
+---
+title: Terrestrial C cycle
+---
+
+Contents:
+
+- [Scientific basis](#simplenbox-science) behind the terrestrial C cycle
+- [Implementation details](#simplenbox-implementation)
+
+# Science {#simplenbox-science}
+
+Hector's land C sink is divided into three pools: vegetation ($C_v$), detritus ($C_d$), and soil ($C_s$).
+These sinks are global by default, but they can be divided into $n$ "biomes" with unique parameters and initial pool sizes.
+
+The total C flux from atmosphere to land at time $t$ ($F_{L}(t)$) is the net of C sequestration by photosynthesis (net primary productivity, $NPP_i(t)$) and C release by heterotrophic respiration ($RH(t)$), summed across each biome $i$:
```

Comment:
It's the net of photosynthesis, RH, and LUC ?

Reply:
Addressed in 5ba2eaf.